docs/backup-restore.md

```diff
@@ -1,32 +1,101 @@
+Otherwise, the time you get as a paused duration might be too different from the time in which the pause was actually conducted, which could restore to a point where ongoing transactions exist.
+Also, it is recommended to pause a long enough time (e.g., 10 seconds) and use the midtime of the paused duration as a restore point since clock synchronization cannot perfectly synchronize clocks between nodes.
+
+## Restore Backup
+
+### JDBC databases
+
+You can restore the backup with your favorite way for JDBC databases.
+You can use `mysql` command in MySQL and `psql` command in PostgreSQL to achieve that.
+If you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL,
+you can restore to any point within the backup retention period with the automated backup feature.
+
+### Cassandra
+
+To restore the backup, you should stop the Cassandra and clean the `data`, `commitlogs`, and `hints` directories then restores the backup copies of all the nodes of the cluster, and start the cluster.
+
+To avoid mistakes when doing restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+You can restore to any point using `CLUSTER-ID` and `SNAPSHOT-ID`.
+Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+### Cosmos DB
+
+To restore a backup, please follow the [azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal) and change the default consistency to `STRONG` after restoring the backup.
+It is highly recommended to use the midtime of the pause as a restore point as we explained earlier.
+
+### DynamoDB
+
+You can restore the table to a point in time using the DynamoDB console or the AWS Command Line Interface (AWS CLI). The point-in-time recovery process restores to a new table.
+By default, the table can only be restored one by one.
+
+You can restore tables one by one from the [Amazon DynamoDB console](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/PointInTimeRecovery.Tutorial.html) using the following steps and use the midtime of the pause as a restore point
+ 
+* Restore with PITR of table A to another table B
+* Take a backup of the restored table B (assume the backup is named backup B)
+* Remove table A
+* Create a table named A with backup B
+
+Note that you need to follow the above steps because the application requires tables with the same name as before. In DynamoDB, table restoration is possible only with an alias, so you can restore the table with alias and delete the actual table and rename the restored table (alias) with an actual name.
```

Comment:
```suggestion
Note that you need to follow the above steps because it assumes the application requires tables with the same names as before. In DynamoDB, a table can only be restored with an alias, so you can restore the table with an alias and delete the original table and rename the alias to the original name.
```

Reply:
Thank you for your review comments.
Document updated based on review comments.